skymp5-server/cpp/papyrus_vm_lib/Reader.cpp

```diff
@@ -13,7 +14,9 @@ void Reader::Read()
   if (File.is_open()) {
 
     File.seekg(0, std::ios_base::end);
+    fl_size = File.tellg();
     File.seekg(0, std::ios_base::beg);
+    arrayBytes.reserve(fl_size);  // Желательно, чтобы ближайшая степень двойки была, но с лимитом в 4096
```

Comment:
1. Sounds more like a self-note, should not go to final commit.
2. No reason to round up to be power of two and/or divisible by 4096, allocators should do that by themselves. If only we care a lot about caching, but no reason to overoptimize stuff here, it requires a more global approach targeting entire project to have any effect.

P.S. Let's also keep all comments, including temporrary ones, in English